client/my-sites/plan-features-2023-grid/lib/get-display-prices.ts

```diff
@@ -0,0 +1,17 @@
+import { formatCurrency } from '@automattic/format-currency';
+
+export function getDisplayPrices( prices: ( number | null )[], currencyCode: string ) {
```

Comment:
Could we think of ways of simplifying this into a simple `number[]`? 

In general arrays perform the best when it's homogeneous. For a batch-like function like this one, having to check whether the value exists before operating on it can actually lead to inefficient code generated. I didn't look, but I suspect this case would be especially true since `number` is a primitive type while `null` is an object. 

I know performance is likely not a concern for this function since the length of `prices` is so short, but I still think it's in general not a good idea to introduce arrays with potential "holes" :)

Reply:
I really like this observation, and it's something I'll keep in mind moving forward 😎

I made the updates in https://github.com/Automattic/wp-calypso/pull/81743/commits/af7062cb7546f62a6bdfdbfd881b64d691bd3751#diff-2e9ac2fdaccb251f8183cec55d715948fa8f6acdf1532236edaa5809c8c882cfR13